File: 240120/연속되는 수 2/continuous-number2.py
n = int(input())

arr = [int(input()) for _ in range(n)]
max_num = 0
cnt = 1
for i in range(n):
    if i == 0 or arr[i] != arr[i-1]:
        cnt = 1
    else:
        cnt += 1
    if cnt >= max_num:
        max_num = cnt

        
print(max_num)

File: 240121/연속되는 수 2/continuous-number2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        int[] array = new int[n];
        int cnt = 1;
        int max_num = 0;
        for (int i = 0; i < n; i++) {
            int num = scan.nextInt();
            array[i] = num;
            if (i == 0 || array[i] != array[i-1]) {
                cnt = 1;
                // System.out.printf("%d", array[i]);
            }
            else {
                cnt += 1;
            }
            if (cnt > max_num) {
            max_num = cnt;
        }
        
        }
        System.out.printf("%d\n", max_num);
    }
}